### PR TITLE
Revert "Alerting: add template funcs"

### PR DESCRIPTION
**Pull request by santihernandezc**
Reverts grafana/grafana#38404 to fix known issues (i.e we're using `*float64` for the value instead of just `float64`)

### CODE REVIEW

**Review by grobinson-grafana — APPROVED** [review]

File: docs/sources/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule.md [hunk 1]
```diff
@@ -112,23 +112,6 @@ The following template variables are available when expanding annotations and la
 | $values | The values of all reduce and math expressions that were evaluated for this alert rule. For example, `{{ $values.A }}`, `{{ $values.A.Labels }}` and `{{ $values.A.Value }}` where `A` is the `refID` of the expression. This is unavailable when the rule uses a classic condition. |
 | $value  | The value string of the alert instance. For example, `[ var='A' labels={instance=foo} value=10 ]`.                                                                                                                                                                                  |
 
-#### Template functions
```
Comment (achatterjee-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule.md:115]: Missing section.

**Review by achatterjee-grafana — COMMENTED** [review]
Docs added are now missing.

File: docs/sources/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule.md (hunk 1 above)
Comment (gerobinson) [reply docs/sources/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule.md:115]: This is a revert?

**Comment by gerobinson** [comment]
We are reverting this PR because it undid https://github.com/grafana/grafana/pull/38766 and we want that to make the 8.1.4 release tomorrow.

File: docs/sources/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule.md (hunk 1 above)
Comment (achatterjee-grafana) [reply docs/sources/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule.md:115]: The table under "Template functions" section was added for the PR and it is relevant information? If yes, we need to add that back.

**Comment by achatterjee-grafana** [comment]
> We are reverting this PR because it undid #38766 and we want that to make the 8.1.4 release tomorrow.

Oh ok - got it.

**Pull request merged**